03/app.js

```diff
@@ -43,10 +43,38 @@ books.getAuthor = function(isbn) {
 
 books.getTitle = function(isbn, lang) {
 
+    const title = this[isbn].title;
+
+    if (!title) {
+        return null;
+    }
+
+
+    if (lang === 'en') {
```

Comment:
Zdecydowanie mało elastyczne. Co jeśli pojawią się inne języki? Lepsza opcja to `if(title[lang])` - jeśli występuje tytuł w danym języki to... np. zwróć :)